File: Pemrograman1/Modul3/Python/PRAK302-2310817310009-Muhamamd_Fauzan_Ahsani.py
nilai = int(input())

if nilai >= 80:
    print("A")
elif nilai <= 79 and nilai >= 70:
    print("B")
elif nilai <= 69 and nilai >= 60:
    print("C")
elif nilai <= 59 and nilai >= 50:
    print("D")
elif nilai < 50:
    print("E")

File: Pemrograman1/Modul3/C/PRAK302-2310817310009-Muhamamd_Fauzan_Ahsani.c
#include <stdio.h>

int main(){
    int nilai;
    scanf("%d", &nilai);

    if(nilai>=80){
        printf("A");
    }
    else if(nilai <= 79 && nilai >=70){
        printf("B");
    }
    else if(nilai <= 69 && nilai >=60){
        printf("C");
    }
    else if(nilai <= 59 && nilai >=50){
        printf("D");
    }
    else if(nilai < 50){
        printf("E");
    }
    return 0;
}